ddl/ddl_api.go

```diff
@@ -1259,9 +1259,8 @@ func (d *ddl) getModifiableColumnJob(ctx context.Context, ident ast.Ident, origi
 	}
 
 	newCol := table.ToColumn(&model.ColumnInfo{
-		ID:                 col.ID,
-		Offset:             col.Offset,
-		State:              col.State,
+		ID: col.ID,
```

Comment:
Please add some comments why we not set offset here.

Reply:
I think to add the comment to line454 in `column.go` is enough.